contributors/devel/development.md

```diff
@@ -461,12 +461,6 @@ run the tests, and then tear the cluster down.
 
 **Note:** Running all E2E tests takes a *very long time*!
 
-To run E2E tests, use this command:
-
-```sh
-make test-e2e
-```
-
```

Comment:
This command is invalid now.
```
$ make test-e2e
make: *** No rule to make target 'test-e2e'.  Stop.
```

Reply:
/cc @dims @BenTheElder 
I somehow completely missed that y'all removed this in https://github.com/kubernetes/kubernetes/pull/84696

This seems like a bad developer experience? What do you two do now to run e2e tests against local development?

Reply:
one way to workaround is building `e2e.test` binary, the binary can still work well with local provider, something like this, 
```
/usr/bin/e2e.test -ginkgo.focus=AdmissionWebhook  -provider=local -report-dir=/var/log/kubernetes/e2e
```

Reply:
Nobody was running this make target? As evidenced by nobody missing it or complaining :-)

e2e tests are run through a _lot_ of different workflows, but hack/e2e.go was terrible.

I personally use the e2e-k8s.sh script from KIND most of the time, but, kubetest, kubetest2, or e2e.test directly are all viable.